docs/operate-and-deploy/migrations-tool.md

```diff
@@ -37,6 +37,8 @@ types of ksqlDB statements:
 * `CREATE TABLE ... AS SELECT`
 * `CREATE OR REPLACE`
 * `INSERT INTO ... AS SELECT`
+* `PAUSE <queryID>`
```

Comment:
Can we also update the java client docs at https://github.com/confluentinc/ksql/blob/0d973038399b95006c3334800fb31ffe587d4a94/ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/Client.java#L135-L136 (and for the other method overload)?

Reply:
Also the java client docs (https://github.com/confluentinc/ksql/blob/master/docs/developer-guide/ksqldb-clients/java-client.md) describing what you can do with `executeStatement()`.